test/Microsoft.AspNet.Mvc.IntegrationTests/ControllerPropertiesIntegrationTest.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNet.Http;
+using Microsoft.AspNet.Mvc.ModelBinding;
+using Xunit;
+
+namespace Microsoft.AspNet.Mvc.IntegrationTests
+{
+    public class ControllerPropertiesIntegrationTest
```

Comment:
I don't see what makes these 'controller property' tests - they look just like the other ones and cover scenarios that are already covered.